draft.py

```diff
@@ -135,6 +135,10 @@ def get_team_list(self):
         sorted_teams = sorted(list(self.team_list))
         return [str(t[0]) + t[1] for t in sorted_teams]
 
+    def get_team_square(self):
```

Comment:
Can you add typing (e.g. `def get_team_square(self) -> List[int]:` and a quick comment explaining what it does? I don't immediately follow it